tests/functional/a/assert_on_tuple.py

```diff
@@ -1,11 +1,11 @@
 '''Assert check example'''
 
 # pylint: disable=comparison-with-itself, comparison-of-constants
-assert (1 == 1, 2 == 2), "no error"
+assert (1 == 1, 2 == 2), "no error" # [assert-on-tuple]
```

Comment:
```suggestion
assert (1 == 1, 2 == 2), "message is raised even when there is an assert message"  # [assert-on-tuple]
```
We need to update the example to account for the new behavior.